tools/generate-integration-docs-screenshot

```diff
@@ -138,6 +138,29 @@ def custom_headers(headers_json: str) -> Dict[str, str]:
         )
 
 
+def send_bot_mock_message(
+    bot: UserProfile, integration: Integration, fixture_path: str, config: BaseScreenshotConfig
+) -> None:
+    # Delete all messages, so new message is the only one it's message group
+    Message.objects.filter(sender=bot).delete()
+    data, _, _ = get_fixture_info(fixture_path)
+
+    client, _ = Client.objects.get_or_create(name="integration_screenshot")
+
+    try:
+        check_send_stream_message(
+            bot, client, integration.stream_name, data["subject"], data["body"]
+        )
```

Comment:
I don't understand why we're doing `check_send_stream_message` here -- which sends a message internally.  I would have thought we'd be just doing an HTTP post to the API -- perhaps via https://zulip.com/api/send-message -- for this simulation.

Reply:
That makes sense. Perhaps I can do it the same way as `send_bot_payload_message`.